frontend/src/components/Nav.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function Nav() {
+  return (
+    <div>Nav</div>
+  )
+}
```

Comment:
Nice, i like how its in the component folder.

Reply:
Easy breezy beautiful